tests/state/test_get_state.py

```diff
@@ -0,0 +1,146 @@
+"""
+Provide tests for command line interface's get state command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+ADDRESS = '0000000000000000000000000000000000000000000000000000000000000000000001'
+
+
+def test_get_state():
+    """
+    Case: get a state by address.
+    Expect: state is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get',
+        '--address',
+        ADDRESS,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert isinstance(json.loads(result.output), dict)
+
+
+def test_get_state_with_invalid_address():
+    """
+    Case: get a state by its invalid address.
+    Expect: the following address is invalid error message.
+    """
+    invalid_address = '044c7db163cf2'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get',
+        '--address',
+        invalid_address,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            'address': [
+                f'The following address `{invalid_address}` is invalid.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_state_without_node_url(mocker):
+    """
+    Case: get a state by address without passing node URL.
```

Comment:
`get a state by address` > `get a state by its address`.

Please, check whole documentation in your PR.

Reply:
@dmytrostriletskyi, done.